stable/burrow/values.yaml

```diff
@@ -1,66 +1,103 @@
 image:
-  repository: quay.io/monax/db
-  tag: 0.17.1
+  repository: hyperledger/burrow
+  tag: 0.23.3
   pullPolicy: IfNotPresent
 
 chain:
-  name: simpleTestChain
-  id: simpleTestChain
+  name: ""
   numberOfNodes: 1
   logLevel: info
   extraSeeds: []
+  testing: false
+
+validatorAddresses:
```

Comment:
let's comment these lines out and add a new file `ci/base.yaml` which includes test addresses and keys it should also flip the `chain.testing` variable to `true`